### DISCUSSION TITLE
Grap'n'move module only when the mouse is between the  screws

### DISCUSSION THREAD

**Discussion by subderground** [DISTRHO/Cardinal#399] [Ideas]
Hi,
I often accidentally move modules when I set parameters.
It would be cool if the "move zone" was restricted to the two areas between the screws (top and bottom of each module).
I know the there is a module lock. ;-) But lock/unlock all the time is not so fun.

I don't know if it's possible and easy to make... That's just an idea.
Have a great day !
T.
![CardinaleGrap-n-move](https://user-images.githubusercontent.com/105162700/193877180-be3917b2-0dbf-47e4-a403-d014182e1071.jpg)

**Comment by dromer** [comment]
I know what you mean, but this really is an upstream issue (with Rack), so best to report there instead :#

**Comment by falkTX** [comment]
Issue is that some modules have content there, which would make them unmovable.
There is a view menu option to lock modules into position, so you dont move them by mistake.

Realistically, there is very little chance of this changing..
